### ISSUE TITLE
[P1] Activation-aware export uses source Hessians after rotating or balancing weights

### ISSUE THREAD

**Issue by RobTand** [RobTand/tessera#231]
Reviewed ea72d23cbf26a453d0d740afac537fb6d6b8afbc. This covers the combination of activation-aware encoding with an input rotation or segment-2a diagonals. The issue was independently traced through the producer and the encoder and is recorded once here.

ActivationSource.for_unit takes the captured source-coordinate H at export.py:622-623, derives LDL from that H at :632-635, and passes H itself or a power of its diagonal as the refit metric at :637-642. The trailing objective has the same behavior. export_checkpoint (:1512-1521) and its streaming equivalent (:1932-1942) simultaneously pass these values and rotation/with_diagonals. encode_linear_planes forwards them unchanged (:1400-1413). encode_unit then rotates W at :2318 and removes diagonals at :2332, but uses the original metric and factor against those transformed coordinates.

Let Wwork = Dv^-1 Wsource R Du^-1, as the implementation defines it. Source-coordinate activations become xwork = Du R^T xsource, so their metric is Hwork = Du R^T Hsource R Du. Source output loss is sum_r sv_r² Ework_r Hwork Ework_r^T. Therefore a general source H is the wrong matrix after rotation or input balancing. An LDL factor cannot generally be reused after this change of basis; it must be factored from the metric in the encoded coordinates. With shared LUT fitting, the output-row sv² factors also affect the shared objective. A diagonal source H normally becomes dense under rotation, so forwarding a diagonal power is not automatically a valid substitute.

A 32-column cheap CPU refit demonstrates that the mismatch is numerically substantive without requiring a model or a quality sweep:

```python
import torch
from tessera.diagonals import hadamard_block
from tessera.scale_channel import refit_channel_scale

R = hadamard_block(32)
Hsource = torch.diag(torch.arange(1, 33).float())
Hwork = R.T @ Hsource @ R
work = torch.ones(1, 32)
units = torch.zeros(1, 32)
units[0, 0] = 1
stored = torch.ones(1, dtype=torch.float16)
for metric in (Hsource, Hwork):
    _, scale = refit_channel_scale(work, units, stored, 1., metric)
    error = work - scale[:, None] * units
    print(scale.item(), ((error @ Hwork) * error).sum().item())
# Untransformed H: scale 1.0, actual Hwork loss 46.499996185302734
# Correct Hwork: scale 0.06060791015625, loss 31.939390182495117
```

Run with PYTHONPATH=src and native math threads at one. This was reproduced on torch 2.10.0+cpu. It is an objective witness for fixed codes, not a served-quality result. The end-to-end reachable combination is established by the producer call chain above.

Current tests/test_ldlq_lut_plane.py validates identity/diagonal/full metric relationships in one coordinate system. tests/test_compensate.py checks rotation in the standalone stitching path, whose callback returns original-coordinate reconstruction before propagation; that is different from encode_unit's internal transformed-coordinate LDL path. The checked tests do not cover the exporter combining captured H with transformed weights.

Fix direction: define and centralize the metric's basis at the producer/encoder boundary. Transform the Hessian (and row weighting where necessary) consistently before LDL factorization and metric refit, with an explicit convention for where regularization is applied; alternatively refuse unsupported combinations at export. Acceptance should compare the same functional quadratic before and after nontrivial rotations/diagonals, cover both exporter variants and trailing objectives, and verify LDL-factor/schedule agreement in the resulting basis. Do not merely change the recorded objective label.

No fix was attempted because the user explicitly requested an audit and issues without implementation changes. Existing measurements were accepted at face value; no new served comparison or promotion claim was made.

Disposition: filed for a separate implementation agent. The user explicitly requested an audit and issues, with no fixes; no implementation changes were made for this finding.
